### PR TITLE
fix! deterministic registration order

### PR DESCRIPTION
**Pull request by octonato**

### CODE REVIEW

File: samples/java-valueentity-customer-registry/src/main/java/customer/Main.java [hunk 1]
```diff
@@ -22,14 +22,22 @@
...
+      CustomerByEmailView::new,
+      CustomerByNameView::new,
+      CustomerSummaryByNameView::new,
+      CustomersResponseByNameView::new);
+      /*
+      // tag::register[]
+      return AkkaServerlessFactory.withComponents(
+        CustomerValueEntity::new,
+        CustomerByNameView::new);
+      // end::register[]
```
Comment (pvlugter) [inline samples/java-valueentity-customer-registry/src/main/java/customer/Main.java:37]: Maybe this one should have its own order, so the code snippet is still part of what gets compiled?

**Review by pvlugter — APPROVED** [review]
looks good

**Review by raboof — APPROVED** [review]

**Review by johanandren — APPROVED** [review]
LGTM

**Comment by johanandren** [comment]
Merging this since I think it is causing me CI problems in another PR.

**Comment by johanandren** [comment]
Refs #407

**Pull request merged**